e2e-test/fluxd.yaml

```diff
@@ -0,0 +1,157 @@
+kind: Secret
+metadata:
+  creationTimestamp: null
+  name: flux-git-deploy
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    name: flux
+  name: flux
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  labels:
+    name: flux
+  name: flux
+rules:
+  - apiGroups: ["*"]
+    resources: ["*"]
+    verbs: ["*"]
+  - nonResourceURLs: ["*"]
+    verbs: ["*"]
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    name: flux
+  name: flux
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: flux
+subjects:
+  - kind: ServiceAccount
+    name: flux
+    namespace: default
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flux-recv
+data:
+  fluxrecv.yaml: |
+    fluxRecvVersion: 1
+    endpoints: []
+---
```

Comment:
This CM could be deleted now, as `flux-recv` is no longer in the deployment?
```suggestion
```

Reply:
definitely! 👍